packages/angular/src/utils/mf/utils.ts

```diff
@@ -89,15 +89,18 @@ export async function getModuleFederationConfig(
     projectGraph = await createProjectGraphAsync();
   }
 
-  if (!projectGraph.nodes[mfConfig.name]?.data) {
+  const NPM_SCOPE_REGEX = new RegExp(/^@{1}(\w)+\/{1}(?!\/)/);
```

Comment:
nit: you can simplify this:
```suggestion
  const NPM_SCOPE_REGEX = /^@\w+\/(?!\/)/;
```

Reply:
Yeah, that's simpler, also made me realise i missed a use case "@my-name/" which this slight modification to what you have suggested solves `^@(\w|\w-\w)+\/(?!\/)`